deps/src/main/java/com/microsoft/azure/sdk/iot/deps/util/Tools.java

```diff
@@ -36,236 +30,4 @@ public static Boolean isNullOrEmpty(String value)
 
         return retVal;
     }
-
-    /**
-     * Helper function to check if the input string is null or contains only whitespace(s)
-     *
-     * @param value The string to check
-     * @return The value true if the input string is empty or contains only whitespace(s)
-     */
-    public static Boolean isNullOrWhiteSpace(String value)
```

Comment:
All of these helper methods were only being called from test code, so there is no reason to keep it